Set version of pyscf/mbd extension and fix imports on init
author: Alex Domingo (Vrije Universiteit Brussel)
diff -Nru pyscf.orig/mbd/__init__.py pyscf/mbd/__init__.py
--- pyscf.orig/mbd/__init__.py	2022-11-25 02:03:32.583625000 +0100
+++ pyscf/mbd/__init__.py	2022-11-25 02:04:11.959827000 +0100
@@ -12,4 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+__version__ = '0.0.1'
+
 from .mbd import mbd_rsscs, bohr
diff -Nru pyscf.orig/mbd/__init__.py pyscf/mbd/__init__.py
--- pyscf.orig/mbd/__init__.py	2022-11-25 02:16:14.580180000 +0100
+++ pyscf/mbd/__init__.py	2022-11-25 02:16:26.123674000 +0100
@@ -14,4 +14,4 @@
 
 __version__ = '0.0.1'
 
-from .mbd import mbd_rsscs, bohr
+from .mbd import mbd_rsscs
